src/epd/GxEPD2_213_BN.h

```diff
@@ -0,0 +1,92 @@
+// ##########################################################
+// #   Meshtastic: Backported from GxEPD v1.5.5 to v1.3.4   #
+// #   (For Wireless Paper V1.0)                            #
+// ##########################################################
+//
+//
+// Display Library for SPI e-paper panels from Dalian Good Display and boards from Waveshare.
+// Requires HW SPI and Adafruit_GFX. Caution: the e-paper panels require 3.3V supply AND data lines!
+//
+// Panel: DEPG0213BN : https://www.dke.top/products/213-eiink-display
+// Controller : SSD1680 : https://www.good-display.com/companyfile/101.html
+// Display: LILYGO® T5 V2.3.1 2.13 inch : https://www.aliexpress.com/item/32869729970.html
+//
+// Author: Jean-Marc Zingg
+//
+// Version: see library.properties
+//
+// Library: https://github.com/ZinggJM/GxEPD2
+
+#ifndef _GxEPD2_213_BN_H_
+#define _GxEPD2_213_BN_H_
+
+#include "../GxEPD2_EPD.h"
+
+class GxEPD2_213_BN : public GxEPD2_EPD
+{
+  public:
+    // attributes
+    static const uint16_t WIDTH = 128;
+    static const uint16_t WIDTH_VISIBLE = 122;
+    static const uint16_t HEIGHT = 250;
+    static const GxEPD2::Panel panel = GxEPD2::DEPG0213BN;
+    static const bool hasColor = false;
+    static const bool hasPartialUpdate = true;
+    static const bool hasFastPartialUpdate = true;
+    static const uint16_t power_on_time = 100; // ms, e.g. 95868us
+    static const uint16_t power_off_time = 250; // ms, e.g. 140350us
+    static const uint16_t full_refresh_time = 8100; // ms, e.g. 4011934us
```

Comment:
This was incorrectly changed during testing. Should be lowered.
See [ZinggJM/GxEPD2](https://github.com/ZinggJM/GxEPD2/blob/220fc5845c08b83c8dbac63e0cb83e1a774071ca/src/epd/GxEPD2_213_BN.h#L32)